devices/HumanStateProvider/HumanStateProvider.cpp

```diff
@@ -1235,7 +1222,7 @@ bool HumanStateProvider::open(yarp::os::Searchable& config)
                  << pImpl->customConstraintMatrix.cols() << ") are not equal";
         return false;
     }
-    yInfo() << "******* DOF: " << modelLoader.model().getNrOfDOFs();
+    //yInfo() << "******* DOF: " << modelLoader.model().getNrOfDOFs();
```

Comment:
this commented line can be removed

Reply:
Fixed in https://github.com/robotology/human-dynamics-estimation/pull/184/commits/be3f54f2629c6bbb61fc5786ae225d1ec88522c6